src/Docker.DotNet/Models/ServiceFilter.cs

```diff
@@ -9,30 +9,30 @@ public class ServicesListParameters
         [QueryStringParameter("filters", false, typeof(MapQueryStringConverter))]
         public ServiceFilter Filters { get; set; }
     }
-
-    public class ServiceFilter : Dictionary<string, string>
+    

```

Comment:
NIT: extra space

Reply:
Haven't managed to get rid of extra space using VSCode